src/Configurator/AbstractConfigurator.php

```diff
@@ -47,12 +47,12 @@ protected function write($messages): void
         $this->io->writeError($messages, true, IOInterface::VERBOSE);
     }
 
-    protected function isFileMarked(Recipe $recipe, $file)
+    protected function isFileMarked(Recipe $recipe, $file): bool
```

Comment:
Shall we typehint the `$file` parameter as `string` as well? According to all concrete configurators, the given `$file` argument is always a file path as a string.